_inc/client/setup-wizard/feature-toggle-group/content.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+
+const featureGroups = {
+	security: {
+		title: __( 'Security' ),
+		details: __(
+			'Keep your site totally backed up, prevent unwanted intrusions, find issues with malware scanning, and stop spammers in their tracks.'
```

Comment:
"totally" feels a bit off to me. Maybe we could have Editorial reviewing the contents at some point to make sure it's in line with our other content in the dashboard?

Reply:
@robertf4 let's remove the word "totally": 
"Keep your site backed up, prevent unwanted intrusions, find issues with malware scanning, and stop spammers in their tracks."